lib/membrane_opus/util.ex

```diff
@@ -66,4 +66,20 @@ defmodule Membrane.Opus.Util do
       _otherwise -> :error
     end
   end
+
+  @spec validate_pts_integrity(list(), any()) :: :ok
```

Comment:
```suggestion
  @spec validate_pts_integrity([Membrane.Buffer.t()], integer()) :: :ok
```
It is better, to be more specific in typespecs, when it is possible